addons/website_event_track/models/event_track.py

```diff
@@ -168,6 +170,21 @@ def _compute_speaker_image(self):
             if not track.image:
                 track.image = track.partner_id.image_256
 
+    @api.depends('partner_name', 'partner_function', 'partner_company_name')
+    def _compute_partner_tag_line(self):
+        for track in self:
+            tag_line = str()
```

Comment:
I think that it is better to initialize the tagline with an empty string to avoid mixing types. It will be less prone to errors. In Python, an empty string is considered as a falsy value: The programmer could check that the tagline is empty as he/she would do with a boolean.

Example:
```
if not tagline:
    return
```